webapp/src/components/user_attribute/index.ts

```diff
@@ -9,6 +9,7 @@ import {GlobalState, pluginStateKey} from 'src/types/store';
 
 import UserAttribute from './user_attribute';
 
+export type UserAttributeProps = UserAttributeStateProps & UserAttributeDispatchProps
```

Comment:
Curious what you think about this for declaring the props?

```ts
type UserAttributeDispatchProps = ReturnType<typeof mapDispatchToProps>;
type UserAttributeStateProps = ReturnType<typeof mapStateToProps>;

export type UserAttributeProps = UserAttributeDispatchProps & UserAttributeStateProps;
```

Reply:
@mickmister That looks good. also decreases the line on code